js/src/components/paid-ads/campaign-preview/campaign-preview.js

```diff
@@ -26,6 +32,9 @@ import './campaign-preview.scss';
 
 /**
  * @typedef { import("./index.js").AdPreviewData } AdPreviewData
+ *
+ * @typedef {Object} CampaignPreviewHandler
+ * @property {(step: number) => void} moveBy Move the currently displayed preview ad by how many steps, e.g. 1 for moving to the next and -1 for the previous.
```

Comment:
💅 I guess `steps` should be more accurate than `step` since ads far as I know it could be more than 1 step movement.

Besides that, since we don't use movements of more than one step.  I guess it could be refactored as `backwards : bool = false` and move the preview one step backwards if `backwards` is true.